maquetado/application.css

```diff
@@ -0,0 +1,4 @@
+.title {
+  color: red; }
+
+/*# sourceMappingURL=application.css.map */
```

Comment:
This it's not necessary, please delete this line

Reply:
This file should be ignered, as all the css files of the proyect.